spec/bgs/services/corporate_update_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'bgs'
+
+describe BGS::CorporateUpdateWebService do
+  let(:poa_code) { '091' }
+  let(:participant_id) { '600043201' }
+  let(:service) do
+    BGS::Services.new(
+      external_uid: 'something',
+      external_key: 'something'
+    )
+  end
+
+  xit 'updates poa access' do
+    pending('Receiving tuxedo error when hitting endpoint')
```

Comment:
have a question out in slack for why i might be getting this error:
https://lighthouseva.slack.com/archives/C01A2353QTV/p1621368853002300

Reply:
ok, this was because we didnt have permissions for the underlying tuxedo service and permissions have been granted for the linktest environment.

this will require getting the correct permissions added when we go to production as well.